src/components/NavHeader.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="banner">
```

Comment:
Same as before, best practice is to not use div if we have another choice. Main, section, article, nav, are some examples of better tags.